releases.md

```diff
@@ -53,21 +97,69 @@ In this release, GoCenter has enriched the search functionality which includes m
 *	Fixed elastic json escape bug
 *	Added support to Artifactory 6.10.x
 *	Added support to modules with .go in their names
+ 
+## 2019 May 24: Go version updates
+This month we’re continuing to make updates to Go Center that incorporate changes from Go 1.12. These include:
+*	store/fetch go.mod-public/suggested in bucket and server via rest when invoked
```

Comment:
details like this can be removed. We should instead talk about the nature of the change.